bitnami/jupyterhub/values.yaml

```diff
@@ -0,0 +1,1592 @@
+## @param commonAnnotations  Annotations to add to all deployed objects
+##
+commonAnnotations: {}
+
+## Extra objects to deploy (value evaluated as a template)
+## @param extraDeploy Array of extra objects to deploy with the release
+##
+extraDeploy: []
+
+## @section Hub deployment parameters
+## Hub deployment parameters
+##
+hub:
+  image:
+    ## @param hub.image.registry Hub image registry
+    ##
+    registry: docker.io
+    ## @param hub.image.repository Hub image repository
+    ##
+    repository: jupyterhub/k8s-hub
+    ## @param hub.image.tag Hub image tag (immutabe tags are recommended)
+    ##
+    tag: 0.11.1
+    ## Specify a imagePullPolicy
+    ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+    ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+    ##
+    ## @param hub.image.pullPolicy Hub image pull policy
+    ##
+    pullPolicy: IfNotPresent
+    ## Optionally specify an array of imagePullSecrets.
+    ## Secrets must be manually created in the namespace.
+    ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+    ##
+    # pullSecrets:
+    #   - myRegistryKeySecretName
+    ## @param hub.image.pullSecrets Hub image pull secrets
+    ##
+    pullSecrets: []
+
+  ## Configure extra options for liveness and readiness probes
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#configure-probes
+  ##
+  ## @param hub.livenessProbe.enabled Enable livenessProbe
+  ## @param hub.livenessProbe.initialDelaySeconds Initial delay seconds for livenessProbe
+  ## @param hub.livenessProbe.periodSeconds Period seconds for livenessProbe
+  ## @param hub.livenessProbe.timeoutSeconds Timeout seconds for livenessProbe
+  ## @param hub.livenessProbe.failureThreshold Failure threshold for livenessProbe
+  ## @param hub.livenessProbe.successThreshold Success threshold for livenessProbe
+  ##
+  livenessProbe:
+    enabled: true
+    initialDelaySeconds: 10
+    periodSeconds: 10
+    failureThreshold: 30
+    timeoutSeconds: 3
+    successThreshold: 1
+
+  ## @param hub.readinessProbe.enabled Enable readinessProbe
+  ## @param hub.readinessProbe.initialDelaySeconds Initial delay seconds for readinessProbe
+  ## @param hub.readinessProbe.periodSeconds Period seconds for readinessProbe
+  ## @param hub.readinessProbe.timeoutSeconds Timeout seconds for readinessProbe
+  ## @param hub.readinessProbe.failureThreshold Failure threshold for readinessProbe
+  ## @param hub.readinessProbe.successThreshold Success threshold for readinessProbe
+  ##
+  readinessProbe:
+    enabled: true
+    initialDelaySeconds: 10
+    periodSeconds: 10
+    failureThreshold: 30
+    timeoutSeconds: 3
+    successThreshold: 1
+
+  ## Hub base URL
+  ## @param hub.baseUrl Hub base URL
+  ##
+  baseUrl: /
+
+  ## Hub admin user
+  ## @param hub.adminUser Hub Dummy authenticator admin user
+  ##
+  adminUser: user
```

Comment:
Is there any reason why you chose "user" as the default value?

Reply:
To comply with Bitnami standards. We normally use user in other applications. In the upstream chart there is no user as they don't set any authenticator